src/modules/history/actions/BorrowRateModeBlock.tsx

```diff
@@ -0,0 +1,44 @@
+import { Trans } from '@lingui/macro';
+import { Typography } from '@mui/material';
+import { formatUnits } from 'ethers/lib/utils';
+import React from 'react';
+
+import { ActionFields, TransactionHistoryItem } from '../types';
+
+export const BorrowRateModeBlock = ({
+  swapBorrowRateTx,
+  from,
+}: {
+  swapBorrowRateTx: TransactionHistoryItem<ActionFields['SwapBorrowRate']>;
+  from?: boolean;
+}) => {
+  if (from && swapBorrowRateTx.borrowRateModeFrom === 'Variable') {
```

Comment:
I think the `borrowRateModeFrom` and `borrowRateModeFrom` come back as the numeric values, so this should be updated to be
```
if (from && swapBorrowRateTx.borrowRateModeFrom === 2) {
...
}
```

Reply:
I double checked and it does come back as a string, here's a sample swapBorrowRateTx: 

```
{
    "id": "11374100:19:0x62d0bbeab1336e25a593f49e53be8ec988ae97d20ccd06ec06b52b0feab0608d:37:37",
    "timestamp": 1606925645,
    "txHash": "0x62d0bbeab1336e25a593f49e53be8ec988ae97d20ccd06ec06b52b0feab0608d",
    "action": "Swap",
    "borrowRateModeFrom": "Variable",
    "borrowRateModeTo": "Stable",
    "variableBorrowRate": "33333347828159348205473884",
    "stableBorrowRate": "56666673914079674102736942",
    "reserve": {
        "symbol": "DAI",
        "decimals": 18,
        "name": "Dai Stablecoin",
        "underlyingAsset": "0x6b175474e89094c44da98b954eedeac495271d0f"
    }
}
```

Reply:
Ohh this is different between v2 and v3, I was trying on Optimism.

v2 shows `Variable` and `Stable` and v3 will have `2` and `1`